test/functional/tools/StaticServer.ts

```diff
@@ -1,51 +1,48 @@
 /// <reference path="wait-on.d.ts" />
 
-import { spawn, ChildProcess } from 'child_process';
-import * as process from 'process';
 import * as waitOn from 'wait-on';
+const configure = require('../../../webpack.config.js');
+
+const webpack = require('webpack');
+const middleware = require('webpack-dev-middleware');
+const express = require('express');
+const app = express();
```

Comment:
Do we want to maybe either use `import` syntax for all the imports other than `configure`, or switch `waitOn` to use `require` just for consistency?

Reply:
Oh, great point...I'll switch them all to `require`.